chandra_aca/star_probs.py

```diff
@@ -9,71 +9,53 @@
 from six.moves import zip
 
 from scipy.optimize import brentq
+import scipy.stats
 import numpy as np
 from Chandra.Time import DateTime
 
+print('USING LOCAL STAR PROBS!!!')
```

Comment:
I assume this is just WIP as I don't see any code that actually determines if this is being called as a local import or from the configured python environment.